src/Services/Utils/UtilsParser.js

```diff
@@ -20,9 +21,50 @@ function currencyConvertParse(json) {
   return json;
 }
 
+function dataTypeParse(json) {
+  json = json['util:ReferenceDataItem'];
+
+  if (Object.prototype.toString.call(json) !== '[object Array]') {
+    throw new UtilsParsingError(json);
+  }
+
+  json = json.map((referenceItem) => {
+    const item = {};
+
+    if (Object.prototype.toString.call(referenceItem.Code) === '[object String]') {
+      item.code = referenceItem.Code;
+    }
+    if (Object.prototype.toString.call(referenceItem.Deprecated) === '[object String]') {
+      if (referenceItem.Deprecated === 'false') {
+        item.deprecated = false;
```

Comment:
simply `item.deprecated = referenceItem.Deprecated === 'false'`, 5 lines vs 1 , let's save some bytes

Reply:
Aah! yes this will save those...